frontend/public/components/app.jsx

```diff
@@ -337,6 +340,51 @@ const CaptureTelemetry = React.memo(function CaptureTelemetry() {
   return null;
 });
 
+const WarningPolicyAlert = React.memo(function WarningPolicyAlert() {
```

Comment:
Use a more semantically accurate name:
```suggestion
const AdmissionWebhookWarningNotifications = React.memo(function WarningPolicyAlert() {
```